setup.py

```diff
@@ -21,7 +21,6 @@
     install_requires=[
         'numpy',
         'spikeextractors',
-        'spikesorters',
```

Comment:
Oh yes.
You catch it!
This was totally wird before when installing from sourcs : the spikesorter package was was installed form pypi anyway.

Reply:
that was not supposed to be there. spiketoolkit doesn't use spikesorters.

Can you check why the tests are failing? I guess it's because the str should be a raw str...

Reply:
my guess is that we try to open an existing file not already close with w+.

Reply:
Or a pathlib stuff that should str.

Reply:
> Or a pathlib stuff that should str.

impossible: https://github.com/SpikeInterface/spikeextractors/blob/master/spikeextractors/baseextractor.py#L222

I think the path needs to be `r'path-to-file'` for windows, even if pathlib takes care of `\\`